php/commands/export.php

```diff
@@ -179,10 +180,18 @@ private function check_post_type( $post_type ) {
 		if ( is_null( $post_type ) )
 			return true;
 
+		$post_type = array_unique( array_filter( explode( ',', $post_type ) ) );
 		$post_types = get_post_types();
-		if ( !in_array( $post_type, $post_types ) ) {
-			WP_CLI::warning( sprintf( 'The post type %s does not exist. Choose "all" or any of these existing post types instead: %s', $post_type, implode( ", ", $post_types ) ) );
-			return false;
+
+		foreach ( $post_type as $type ) {
+			if ( ! in_array( $type, $post_types ) ) {
+				WP_CLI::warning( sprintf(
+					'The post type %s does not exist. Choose "all" or any of these existing post types instead: %s',
```

Comment:
I think this syntax needs to be `any` instead of `all`. And, if I provide `any`, it should skip this check.